moped-editor/src/views/projects/projectView/ProjectFiles.js

```diff
@@ -93,23 +93,25 @@ const ProjectFiles = props => {
    * Persists the file data into the database
    * @param {Object} fileDataBundle - The file bundle as provided by the FileUpload component
    */
-  const handleClickSaveFile = fileDataBundle => {
+  const handleClickSaveFile = (fileDataBundle) => {
     createProjectFileAttachment({
       variables: {
         object: {
           project_id: projectId,
           file_name: fileDataBundle?.name,
           file_type: fileDataBundle?.type,
-          file_description: fileDataBundle.description,
-          file_key: fileDataBundle.key,
+          file_description: fileDataBundle?.description,
+          file_key: fileDataBundle?.key,
           file_size: fileDataBundle?.file?.fileSize ?? 0,
+          file_url: fileDataBundle?.url,
           created_by: getDatabaseId(user),
```

Comment:
i got an error when working on this locally, because my database id is 46, but the seed data only has goes up to 3. changing this to 1 gets around the error to be able to save the urls locally.